### ISSUE TITLE
For some reason, Ozai can't go on a spiritual journey?

### ISSUE THREAD

**Issue by jttheghost** [Rystic/ATLA-restored#50]
From Steam

**Comment by OkieMatthewHarrison** [comment]
This is a bigger issue. This is caused by the global flag "HYW_0" being set, meaning no one can go on a spirtual journey once the one hundred year war is declared. After disscussion with "RadicalSquirrel and Cirdain" we concluded that that may be because "everyone's too busy fighting the damn war to go travelling".
We aslo discovered that there is currently no way in the game files where "HYW_0" is cleared. If this is wrong then it is my mistake.
So there needs to be disscussion by the Core Mod Team to decide if people shouldn't go on spiritual journeys after the HYW starts (maybe make the journey more risky, fire nation soldiers, more bandits, etc.) and where to add a clr_flag statement for when the war is over.

TLDR; This may be a feature and not a bug and there is currently no way to clear the global flag "HYW_0" which blocks everyone from have a spritual journey
